cltk/lemmatize/latin/unsupervised.py

```diff
@@ -0,0 +1,382 @@
+        path = expanduser(relativepath)
+        pickle_file = join(path, filename)
+        pickle_file = open(pickle_file, 'rb')
+        self.count_dictionary = pickle.load(pickle_file)
+
+    def train_model(self):
+        '''open all the tesserae files and call read_files() on each to build freq model'''
+        relativepath = join('~', 'cltk_data',
+                            'latin', 'text',
+                            'latin_text_tesserae_collection', 'la')
+        path = expanduser(relativepath)
+        onlyfiles = [f for f in listdir(path) if isfile(join(path, f)) and 'augustine' not in f and 'ambrose' not in f and 'jerome' not in f and 'tertullian' not in f and 'eugippius' not in f and 'hilary' not in f] # pylint: disable=line-too-long
+        onlyfiles = [join(path, f) for f in onlyfiles]
+        for filename in onlyfiles:
+            if '.tess' in filename:
+                print(filename)
+                self.read_files(filename)
+
+    def test_count_dictionary(self, token_list, lemma_list):
+        '''Test the ability of lemmatize(), (which uses the self.count_dictionary dictionary,
+        to predict the most likely lemmatization in ambiguous cases. Punctuation is
+        automatically counted as correct, because the 'punc' lemmatization usage is inconsistent
+        in the test corpus.
+        dependencies
+        ------------
+        itemgetter class from operator package
+        parameters
+        ----------
+        token_list: a list of tokens
+        lemma_list: a list of corresponding 'correct' lemmatizaitons
+        results
+        -------
+        prints four numbers: the number of correctly assigned lemmas in ambiguous cases;
+        the number of ambiguous cases in total; the number of tokens analyzed; and a
+        decimal between 0 and 1 representing the proportion of correct lemmatizations.
+        return
+        ------
+        a list object containing all incorrect lemmatizations for analysis. Format:
+        [(token, answer_given, correct_answer), (token...)]
+
+        NOTE: Initial tests show roughly 91% accuracy, identification of punctuation included.
+        '''
+        trials = 0
+        correct = 0
+        errors = []
+        for position in range(0, (len(token_list)-1)):
+            lemmalist = self.lemmatizer.lookup(token_list[position])
+            lemmalist = lemmalist[1]
+            lemma = max(lemmalist, key=itemgetter(1))
+            if len(lemmalist) > 1:
+                trials = trials + 1
+                if lemma[0] == lemma_list[position] or lemma[0] == 'punc':
+                    correct = correct + 1
+                else:
+                    errors.append((token_list[position], lemma[0], lemma_list[position]))
+        print(correct)
+        print(trials)
+        print(len(lemma_list))
+        rate = (len(lemma_list) - trials + correct) / len(lemma_list)
+        print(rate)
```

Comment:
With this chain of print commands, I would prefer that they be provided as:
"label" : value.

However, more typically, in API library design it's better to return a large tuple of many different items, since the print statements may be swallowed on a remote server, another process, etc.

I like that you return the errors collection, but why not expose the other elements also?